### ISSUE TITLE
I'm unclear on what -locateCmd does

### ISSUE THREAD

**Issue by wwalker** [wvanlint/twf#11]
```
-locateCmd <str>: The command whose output will be interpreted as a path to locate in the file tree.
```
An example would be helpful.

I created a script `aaa` ran:

```
twf -locateCmd aaa
```
3 times. once each outputting:
```
/home/wwalker/bin/aaa
```
```
/bin/aaa
```
```
bin/aaa
```
in no case did it show the file in the tree.

However, this may be a side effect of #9 .

**Comment by wvanlint** [comment]
If you're in the root of this project for example, running `twf --locateCmd "echo internal/filetree/filetree.go"` and pressing `/` should highlight that file now (possibly fixed after #9).

However, `--locateCmd` is most useful if it is an interactive command that can search for files in a complementary way than `twf`, such as fuzzy searching provided by `fzf`.

**Comment by wwalker** [comment]
That explanation makes it clear.  I'll draft some verbiage and put in a PR.

However, it doesn't do that.  It is not running the locateCmd.
First I tried exactly what you show above.  Then in case it was a shell problem, I made an executable (aaa) and ended up verifying that it is not getting run:
```
$ cat ~/bin/aaa
#!/bin/bash

#echo /home/wwalker/bin/aaa
date > ~/tmp/aaa-run
echo internal/state/state.go
$ which aaa
~/bin/aaa
$ cmd/twf/twf --locateCmd aaa
$ ls -l ~/tmp/aaa-run
ls: cannot access '/home/wwalker/tmp/aaa-run': No such file or directory
$ aaa
internal/state/state.go
$ ls -l ~/tmp/aaa-run
-rw-r--r-- 1 wwalker wwalker 32 2020-06-14T21:27:33.048263145 /home/wwalker/tmp/aaa-run
$
```

**Comment by wwalker** [comment]
Created a PR with a minor clarification to the locateCmd.

**Comment by wvanlint** [comment]
Merged #15, thanks for contributing!
